src/main/java/ca/corefacility/bioinformatics/irida/service/remote/ProjectSynchronizationService.java

```diff
@@ -366,12 +362,51 @@ public List<ProjectSynchronizationException> syncSample(Sample sample, Project p
 
 		//list the pairs from the remote api
 		List<SequenceFilePair> sequenceFilePairsForSample = pairRemoteService.getSequenceFilePairsForSample(sample);
-		
+
+		//list the single files from the remote api
+		List<SingleEndSequenceFile> unpairedFilesForSample = singleEndRemoteService.getUnpairedFilesForSample(sample);
+
+		//list the fast5 files from the remote api
+		List<Fast5Object> fast5FilesForSample;
+		try {
+			fast5FilesForSample = fast5ObjectRemoteService.getFast5FilesForSample(sample);
+		} catch (LinkNotFoundException e) {
+			logger.warn("The sample on the referenced IRIDA doesn't support fast5 data: " + sample.getSelfHref());
+			fast5FilesForSample = Lists.newArrayList();
+		}
+
+		//get a list of all remote sequencingObject URLs in the sample
+		Set<String> remoteObjectUrls = new HashSet<String>();
+		remoteObjectUrls.addAll(sequenceFilePairsForSample.stream()
+				.map(pair -> pair.getRemoteStatus().getURL())
+				.collect(Collectors.toSet()));
+		remoteObjectUrls.addAll(unpairedFilesForSample.stream()
+				.map(file -> file.getRemoteStatus().getURL())
+				.collect(Collectors.toSet()));
+		remoteObjectUrls.addAll(fast5FilesForSample.stream()
+				.map(fast5Object -> fast5Object.getRemoteStatus().getURL())
+				.collect(Collectors.toSet()));
+
+		//get a list of all remote sequencingObject URLs in the sample
```

Comment:
This comment should read `all local` instead of `all remote`

Reply:
Fixed in [ea085f1](https://github.com/phac-nml/irida/pull/1345/commits/ea085f15ea3121daffc8ff01fd711cfc432fb29a)